From c82c1b1869ec70bea09d993e5f775fa3ab69574b Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 2 May 2014 14:37:47 -0400
Subject: [misc] mei: amthif: fix checkpatch error

Message-id: <1399041470-21330-75-git-send-email-prarit@redhat.com>
Patchwork-id: 79357
O-Subject: [RHEL7.1 PATCH BZ 1088953 74/77] mei: amthif: fix checkpatch error
Bugzilla: 1088953
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088953

commit 34e267d0f49ef4c54eb7b2abbb1b001fa6f26f0e
Author: Tomas Winkler <tomas.winkler@intel.com>
Date:   Sun Mar 16 14:35:57 2014 +0200

    mei: amthif: fix checkpatch error

    ERROR: else should follow close brace '}'
    + }
    + else {

    Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/mei/amthif.c b/drivers/misc/mei/amthif.c
index 7057291..396f111 100644
--- a/drivers/misc/mei/amthif.c
+++ b/drivers/misc/mei/amthif.c
@@ -248,8 +248,7 @@ int mei_amthif_read(struct mei_device *dev, struct file *file,
  if (copy_to_user(ubuf, cb->response_buffer.data + *offset, length)) {
   dev_dbg(&dev->pdev->dev, "failed to copy data to userland\n");
   rets = -EFAULT;
- }
- else {
+ } else {
   rets = length;
   if ((*offset + length) < cb->buf_idx) {
    *offset += length;
-- 
1.7.1